File: app/src/main/java/com/digicoffer/lauditor/Dashboard/DahboardModels/PracticeHeadModels/AverageBillingRateModel.java
package com.digicoffer.lauditor.Dashboard.DahboardModels.PracticeHeadModels;

public class AverageBillingRateModel {
    String averageBillingRate;
    String CurrencySymbol="";
    String CurrencyCode="";
    public AverageBillingRateModel(String averageBillingRate,String CurrencySymbol, String CurrencyCode) {
        this.averageBillingRate = averageBillingRate;
        this.CurrencySymbol=CurrencySymbol;
        this.CurrencyCode=CurrencyCode;
    }

    public String getCurrencyCode() {
        return CurrencyCode;
    }

    public String getCurrencySymbol() {
        return CurrencySymbol;
    }

    public String getAverageBillingRate() {
        return averageBillingRate;
    }

    public void setAverageBillingRate(String averageBillingRate) {
        this.averageBillingRate = averageBillingRate;
    }
}

File: app/src/main/java/com/digicoffer/lauditor/Dashboard/DahboardModels/PracticeHeadModels/AverageBillingRateModel.kt
package com.digicoffer.lauditor.Dashboard.DahboardModels.PracticeHeadModels

class AverageBillingRateModel(
    var averageBillingRate: String?,
    @get:JvmName("getCurrencySymbol") val currencySymbol: String?,
    @get:JvmName("getCurrencyCode") val currencyCode: String?
)
